components/React.vue

```diff
@@ -76,9 +76,7 @@ const renderReact = async () => {
       continue
     }
     const val = attrs[key]
-    if (typeof val !== 'object' || val === null) {
-      cleanProps[key] = val
-    }
+    cleanProps[key] = val
```

Comment:
The original code was filtering out all objects to avoid passing Vue-specific objects (like `VNodes`) from `attrs` to React.

We can update the filter to allow primitives, null, arrays and objects, but only exclude Vue `VNodes`.

how about something like this?
```suggestion
    const isVNode = typeof val === 'object' && val !== null && '__v_isVNode' in val && (val as any).__v_isVNode
    if (!isVNode) {
      cleanProps[key] = val
    }
```